.gitattributes

```
@@ -1 +1,4 @@
 *.sol linguist-language=Solidity
+prerelease.lock export-ignore
```

Isn't this file now inside `cmake/` subdir?
```suggestion
/cmake/prerelease.lock export-ignore
```

---

Also, the other files might need the `/` at the beginning if paths here work like in `.gitignore`.